From 182549384d437cec1c0b8672b5c5eeb72d64e0ef Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 4 Jun 2023 18:16:49 +0300
Subject: [PATCH] lx2162-clearfog: fix phy-mode for dpmacs 3&4

dpmac 3+4 are recognised by nxp dpaa driver only as 10g rate interfaces
(XGMII), not 2.5/5/10G (usxgmii).
Set the correct phy-mode so that dpaa devices and device-tree nodes can
match, enabling mac address fixup.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm/dts/fsl-lx2162a-clearfog.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/fsl-lx2162a-clearfog.dts b/arch/arm/dts/fsl-lx2162a-clearfog.dts
index b23a578d7e..2b6ea0a206 100644
--- a/arch/arm/dts/fsl-lx2162a-clearfog.dts
+++ b/arch/arm/dts/fsl-lx2162a-clearfog.dts
@@ -168,7 +168,7 @@
 	status = "okay";
 	managed = "in-band-status";
 	sfp = <&sfp_at>;
-	phy-mode = "usxgmii";
+	phy-mode = "xgmii";
 };
 
 /* lower 10G connector */
@@ -176,7 +176,7 @@
 	status = "okay";
 	managed = "in-band-status";
 	sfp = <&sfp_ab>;
-	phy-mode = "usxgmii";
+	phy-mode = "xgmii";
 };
 
 &dpmac5 {
-- 
2.35.3

